crates/sp-domains-fraud-proof/src/bundle_equivocation.rs

```diff
@@ -0,0 +1,247 @@
+//! Module to check bundle equivocation and produce the Equivocation fraud proof.
+//! This is mostly derived from the `sc_consensus_slots::aux_schema` with changes adapted
+//! for Bundle headers instead of block headers
+
+use crate::fraud_proof::{BundleEquivocationProof, FraudProof};
+use codec::{Decode, Encode};
+use sc_client_api::backend::AuxStore;
+use sp_api::{BlockT, HeaderT};
+use sp_blockchain::{Error as ClientError, Result as ClientResult};
+use sp_consensus_slots::Slot;
+use sp_domains::SealedBundleHeader;
+use sp_runtime::traits::NumberFor;
+use std::sync::Arc;
+use subspace_runtime_primitives::Balance;
+
+const SLOT_BUNDLE_HEADER_MAP_KEY: &[u8] = b"slot_bundle_header_map";
+const SLOT_BUNDLE_HEADER_START: &[u8] = b"slot_bundle_header_start";
+
+/// We keep at least this number of slots in database.
+const MAX_SLOT_CAPACITY: u64 = 1000;
```

Comment:
Plz notice, different from consensus block equivocation, the number of bundle header need to store maybe much larger when the `instantiate_domain` call is open to permissionless.

Reply:
Size of SealedBundleHeader is 576 bytes
Assuming there are 1000 domains and bundle header per slot per domain, this will be around `0.55 MB` per slot
In the worst case where 2 operators wins the slot for all domains, then this would be `1.1 MB`
In the worst case where 3 operators wins the slot for all domains, then this would be `1.6 MB`

Taking the worst case for all the slots that are stored for all 1000 domains, this will come to 3.1 GB
At the moment, with 10 domains and worst case scenario,  this will come to around `16.87 KB`

I do not have a big concern here for now and would prefer keeping the constants as defined from substrate.
Though @nazar-pc may have other thoughts on this ?

Reply:
When `instantiate_domain` is permissionless, we don't really have a limit to the number of domains, thus it can be much larger than 1000 domains. I don't have immediate concerns either, but plz add some comment/TODO for revisiting it later.